core/src/main/java/com/zextras/carbonio/files/netty/HttpRoutingHandler.java

```diff
@@ -62,6 +66,12 @@ protected void channelRead0(
     HttpRequest request
   ) {
 
+    if (Endpoints.METRICS.matcher(request.uri()).matches()) {
+      context.pipeline().addLast("metrics-handler", metricsController);
+      context.fireChannelRead(request);
```

Comment:
Add the ExceptionsHandler in order to handle RuntimeExceptions

`.addLast("exceptions-handler", exceptionsHandler);`